web/test/vars-cleaner.test.ts

```diff
@@ -0,0 +1,62 @@
+import { equal } from 'node:assert'
+import { beforeEach, test } from 'node:test'
+import postcss from 'postcss'
+
+import {
+  getVarsCleanerError,
+  resetCleanerGlobals,
+  varsCleaner
+} from '../postcss/vars-cleaner.js'
+
+function run(input: string, output: string): string | undefined {
+  let result = postcss([varsCleaner]).process(input, { from: undefined })
+  equal(result.css, output)
+
+  return getVarsCleanerError()
+}
+
+beforeEach(() => {
+  resetCleanerGlobals()
+})
+
+test('clean unused palette colors', () => {
+  run(
```

Comment:
Let’s test return value here too.

Reply:
I didn't notice that I had left an unused variable that should be returned with an error. I removed it and checked that we should get an undefined value if there are no errors